src/main/resources/fxml/oauth_controller.fxml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 Esri.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.geometry.Insets?>
+<StackPane xmlns:fx="http://javafx.com/fxml" fx:controller="com.esri.samples.portal.oauth.OAuthController"
+           stylesheets="/css/style.css">
+    <HBox>
+        <VBox styleClass="panel-region" spacing="5" maxWidth="150">
+            <padding><Insets left="5" top="5" bottom="5" right="5"/></padding>
+            <Label text="Portal URL" style="-fx-font-weight: bold"/>
+            <TextField fx:id="portalURL" promptText="http://csf.maps.arcgis.com"/>
+            <Label text="Client Id" style="-fx-font-weight: bold"/>
+            <TextField fx:id="clientId" promptText="1EnyBzSblxW"/>
+            <Label text="Redirect URI" style="-fx-font-weight: bold"/>
+            <TextField fx:id="redirectUri" promptText="urn:ietf:wg:oauth:2.0:oob"/>
+            <Button text="Sign In" onAction="#handleSignIn"/>
+        </VBox>
+        <VBox spacing="10" maxWidth="300">
```

Comment:
Could add some not editable text fields below each were information is going to be displayed. I believe it looks a little better.
![oauth](https://cloud.githubusercontent.com/assets/14575538/21424024/06bab9ce-c838-11e6-88d0-1fca527fea23.PNG)